index.js

```diff
@@ -1,13 +1,83 @@
 // TODO: Include packages needed for this application
+const inquirer = require("inquirer");
+const generateMarkdown = require("./utils/generateMarkdown");
+const fs = require("fs");
 
 // TODO: Create an array of questions for user input
```

Comment:
remove these TODO comments please - these should only be present if something has not been completed